config.js

```
@@ -0,0 +1,29 @@
+var chalk = require('chalk');
+
+function getChalkColors(configInput) {
+  // make the read file understandable to chalk.
+  var chalkColors = {};
+  for (var color in configInput.colors) {
+  	if (configInput.colors.hasOwnProperty(color)) {
+  		var colorFunc = chalk;
+      //build up the chalk function
+      configInput.colors[color].forEach(function(style) {

```

Not sure what behaviour you want if this loop throws. Should we (In order of preference): 

1. Throw a nice error to the user saying the color they specified is invalid.
2. Log and simply use the default color
3. Fail silently and use the default color

---

Lets throw a nice little error saying the color they specified is invalid and use the default color. 
So that even if the `lmeconfig.json` file contains invalid colors,

1. user can see the output message he passed to `lme`. (i.e., the basic functionality is not blocked by the thrown error.)
2. user gets a clear understanding of why his color is not showing up.

Correct?

---

Sounds good to me... Will fix up tomorrow :)

---

Yup. Take your time.